doc/api/errors.md

```diff
@@ -128,12 +128,12 @@ they are thrown *after* the calling code has already exited.
 Developers must refer to the documentation for each method to determine
 exactly how errors raised by those methods are propagated.
 
-### Node.js style callbacks
+### Error-first style callbacks
 
 <!--type=misc-->
 
 Most asynchronous methods exposed by the Node.js core API follow an idiomatic
-pattern  referred to as a "Node.js style callback". With this pattern, a
+pattern  referred to as an "error-first style callback". With this pattern, a
```

Comment:
Super-tiny nit here, and I know this was in the original text and not something you introduced, so feel free to ignore this, but it would be good to replace the quotation marks in this line with italics because that's the standard typography for words-as-words. (See https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Text_formatting#Words_as_words for more information if curious.)

Reply:
👍 no problem